week-8/7-destructuring.js

```diff
@@ -0,0 +1,55 @@
+// Destructure this object literal
```

Comment:
I know that "object literal" is the correct jargon but we haven't really been using that - just "object". Happy to keep it as-is if you feel strongly about it, but I would change it